tests/e2e/test_pvc_creation_performance.py

```diff
@@ -41,11 +43,55 @@ def test_pvc_creation_measurement_performance(self, teardown_factory):
         """
         log.info('Start creating new PVC')
 
-        pvc_obj = create_pvc(sc_name=self.sc_obj.name, size=self.pvc_size)
+        pvc_obj = helpers.create_pvc(
+            sc_name=self.sc_obj.name, size=self.pvc_size
+        )
         teardown_factory(pvc_obj)
-        create_time = measure_pvc_creation_time('CephBlockPool', pvc_obj.name)
+        create_time = helpers.measure_pvc_creation_time(
+            self.interface, pvc_obj.name
+        )
         if create_time > 1:
-            raise AssertionError(
+            raise ex.PerformanceException(
                 f"PVC creation time is {create_time} and greater than 1 second"
             )
         logging.info("PVC creation took less than a 1 second")
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    @polarion_id('OCS-1620')
+    @bugzilla('1741612')
+    def test_multiple_pvc_creation_measurement_performance(
+        self, teardown_factory
+    ):
+        """
+        Measuring PVC creation time of 120 PVCs in 60 seconds
+        """
+        number_of_pvcs = 120
+        log.info('Start creating new 120 PVCs')
+
+        pvc_objs = helpers.create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=self.pvc_size,
+            wait=False
+        )
+        for pvc_obj in pvc_objs:
+            teardown_factory(pvc_obj)
+        for pvc_obj in pvc_objs:
+            helpers.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
+        start_time = helpers.get_start_creation_time(
+            self.interface, pvc_objs[0].name
+        )
+        end_time = helpers.get_end_creation_time(
+            self.interface, pvc_objs[number_of_pvcs - 1].name,
```

Comment:
@RazTamir is this sequential creation where we wait for one PVC to be BOUND before sending 2nd request.

If not, then, in scenarios where multiple pvc creations are sent in parallel, the order of them getting BOUND may vary. i.e. PVC97 may get BOUND before PVC95 and so on. So end_time may not always be the time when the 120th PVC got BOUND here. 

Just an observation. Please correct me if I am wrong.

Reply:
You are right @nehaberry .
For now, this test is blocked on BZ - we'll need to figure out how to handle it as soon it is fixed